src/containers/billing/add-card-form.js

```diff
@@ -78,7 +112,34 @@ export default class AddCardForm extends Component {
     addCard: PropTypes.func.isRequired
   };
 
-  handleCardSubmit() {
+  handleCardSubmit(e) {
+    e.preventDefault();
+
+    const {
+      ccNumber,
+      ccExp,
+      cvv
+    } = this.props.fields;
+
+    const [
+      exp_month,
+      exp_year
+    ] = ccExp.value.split('/');
+
+    // TODO: factor payment-processor-specific code out!
```

Comment:
A lot of these TODOs are connected and I find if useful to see them as you are refactoring because it may be trivial to pull in an existing TODO as opposed to having to get back in there later again... Not opposed to creating an issue for some of these that make sense to have issues for tho.